lib/datadog/core/crashtracking/component.rb

```diff
@@ -9,6 +9,14 @@
 module Datadog
   module Core
     module Crashtracking
+      RUNTIME_STACKS_FAILURE =
+        begin
+          require_relative 'crashtracking_runtime_stacks'
+          nil
+        rescue LoadError => e
+          e.message
+        end
```

Comment:
I'm not the biggest fan of this... Specifically -- it creates a new situation in the application where the profiling native extension is loaded (including many of its classes), but the ruby-level code of profiling isn't loaded.

The `profiling/load_native_extension` file was meant to be used by profiling, not for things outside of profiling.

My suggestion would be to instead use `Datadog::Profiling.supported?` / `Datadog::Profiling.unsupported_reason`. These are the public APIs that profiling exposes to the outside, and that let profiling still maintain control of loading of the extension.